src/config/config.cc

```diff
@@ -233,6 +233,7 @@ Config::Config() {
       {"log-retention-days", true, new IntField(&log_retention_days, -1, -1, INT_MAX)},
       {"persist-cluster-nodes-enabled", false, new YesNoField(&persist_cluster_nodes_enabled, true)},
       {"redis-cursor-compatible", false, new YesNoField(&redis_cursor_compatible, true)},
+      {"redis-databases", true, new IntField(&redis_databases, 0, 0, 10000)},
```

Comment:
Is 10000 suitable? not sure what's the limit in redis.

Reply:
cc @git-hulk

Reply:
Yes, it's INT_MAX(2^31 - 1) in Redis open source version. Perhaps we should follow this value.